Tests/SourceKitLSPTests/CodeActionTests.swift

```diff
@@ -425,4 +425,51 @@ final class CodeActionTests: XCTestCase {
 
     try await fulfillmentOfOrThrow([editReceived])
   }
+
+  func testCodeActionForFixItsProducedBySwiftSyntax() async throws {
+    let ws = try await MultiFileTestWorkspace(files: [
+      "test.swift": "protocol 0️⃣Multi 1️⃣ident 2️⃣{}",
```

Comment:
Nitpick similar to the PR in swift-syntax: We usually start counting the location markers at 0. Would be nice to be consistent here.